examples/slash_commands/handlers.go

```diff
@@ -0,0 +1,288 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+func Login(ds *discordgo.Session, r *discordgo.Ready) {
+	log.Printf("Logged in as: %v#%v", ds.State.User.Username, ds.State.User.Discriminator)
+}
```

Comment:
I think having anonymous handler would be better in this case, since it's just oneliner.